IdentityCore/tests/automation/ui_tests_lib/XCUIElement+CrossPlat.m

```diff
@@ -39,8 +39,8 @@ - (void)msidPasteText:(NSString *)text application:(__unused XCUIApplication *)a
 #if TARGET_OS_IPHONE
     [self typeText:text];
 #else
-    [[NSPasteboard generalPasteboard] declareTypes:[NSArray arrayWithObject:NSStringPboardType] owner:nil];
-    [[NSPasteboard generalPasteboard] setString:text forType:NSStringPboardType];
+    [[NSPasteboard generalPasteboard] declareTypes:[NSArray arrayWithObject:NSPasteboardTypeString] owner:nil];
```

Comment:
Why are those changes required? A more detailed PR description would be beneficial

Reply:
In macOS 10.15 the `NSStringPboardType` enum value has been replaced with `NSPasteboardTypeString`

Reply:
I've moved them to the 10.15 PR because that's where they actually belong